rllib/algorithms/bc/bc_catalog.py

```diff
@@ -95,6 +95,7 @@ def build_pi_head(self, framework: str) -> Model:
             hidden_layer_activation=self.pi_head_activation,
             output_layer_dim=required_output_dim,
             output_layer_activation="linear",
+            log_std_clip_param=self._model_config_dict["log_std_clip_param"],
```

Comment:
What if the user doesn't define this in `model_config_dict`?

Reply:
Then the default sets in doesn't it? (`rllib/models/catalog.py`)